src/main/kotlin/com/criptext/mail/scenes/settings/devices/DeviceHolder.kt

```diff
@@ -15,32 +15,32 @@ class DeviceHolder(val view: View) : RecyclerView.ViewHolder(view) {
     private val textViewCurrent: TextView
     private val deviceLayout: FrameLayout
     private val imageDeviceType: ImageView
+    private val imageTrashDevice: ImageView
 
 
     init {
         textViewDeviceName = view.findViewById(R.id.textViewDeviceName) as TextView
         textViewCurrent = view.findViewById(R.id.textViewCurrentDevice) as TextView
         deviceLayout = view.findViewById(R.id.deviceItem) as FrameLayout
         imageDeviceType = view.findViewById(R.id.imageViewDeviceType) as ImageView
+        imageTrashDevice = view.findViewById(R.id.imageViewTrashDevice) as ImageView
     }
 
     fun bindDevice(deviceItem: DeviceItem){
         textViewDeviceName.text = deviceItem.friendlyName
         if(deviceItem.isCurrent) {
             textViewCurrent.setText(R.string.current_device)
-            deviceLayout.isLongClickable = false
+            imageTrashDevice.visibility = View.GONE
```

Comment:
you should put `...visibility  = View.VISIBLE` in the else condition. Remember that the views are always recycling itself.